scripts/stress-build

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
@wvi @honzajavorek This script is running the `npm test` in a `while true` cycle and it stops if the test suite fails. It's for finding flaky tests.